greymatter/core/edge.cue

```diff
@@ -6,80 +6,90 @@
 
 package greymatter
 
-let EgressToRedisName = "\(defaults.edge.key)_egress_to_redis"
+let EgressToRedisName = "\(edge.key)_egress_to_redis"
 
 // Uncomment the below line for use with a remote JWKS provider (in this case, Keycloak)
-// let EdgeToKeycloakName = defaults.edge.oidc.jwt_authn_provider.keycloak.remote_jwks.http_uri.cluster
+// let EdgeToKeycloakName = edge.oidc.jwt_authn_provider.keycloak.remote_jwks.http_uri.cluster
 
 Edge: {
-	name:   defaults.edge.key
-	config: edge_config
-}
+	name: edge.key
+	config: [
```

Comment:
I like this style of writing the config straight in the `config` array

Reply:
Yup, I think you started that for the grocerylist services.